omnibus/files/private-chef-cookbooks/private-chef/recipes/opscode-erchef.rb

```diff
@@ -33,6 +33,8 @@
 
 rabbitmq = OmnibusHelper.new(node).rabbitmq_configuration
 
+elastic_search_version = OmnibusHelper.new(node).elastic_search_major_version
```

Comment:
Figured it out.  @lancewf 's editor had been trolling him the entire time we coded this up.  It put in 10 tabs or something instead of 20 spaces and I'm fairly sure github uses 4 or 8 space tabs.